features/lwipstack/LWIPInterface.cpp

```diff
@@ -273,6 +273,52 @@ char *LWIP::Interface::get_interface_name(char *buf)
     return buf;
 }
 
+static bool convert_lwip_addr_to_mbed(nsapi_addr_t *out, const ip_addr_t *in)
```

Comment:
Why these have been moved from `lwip_tools.cpp`?

Reply:
in older version of mbed-os this function was not available in lwit_tools.cpp. I have re-based the patch and removed this function from LWIPInterface.cpp file.